src/android/IoniCordovaRTMPandHLS.java

```diff
@@ -0,0 +1,75 @@
+package cordova-plugin-ionicrtmphls;
+
+import org.apache.cordova.CordovaPlugin;
+import org.apache.cordova.CallbackContext;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * This class echoes a string called from JavaScript.
+ */
+public class IoniCordovaRTMPandHLS extends CordovaPlugin {
+
+    @Override
+    public boolean execute(String action, JSONArray args, CallbackContext callbackContext) throws JSONException {
+        switch(action) {
+            case "coolMethod":
+                String message = args.getString(0);
+                this.coolMethod(message, callbackContext);
+                return true;
+            case "previewCamera":
+                this.previewCamera(callbackContext);
+                return true;
+            case "swapCamera":
+                this.swapCamera(callbackContext);
+                return true;
+            case "startBroadcasting":
+                this.startBroadcasting(callbackContext);
+                return true;
+            case "stopBroadcasting":
+                this.stopBroadcasting(callbackContext);
+                return true;
+            case "viewLiveStream":
+                this.viewLiveStream(callbackContext);
+                return true;
+            case "requestPermissions":
+                this.requestPermissions(callbackContext);
+                return true;
+        }
+        return false;
```

Comment:
Not require - but I think we can put this on switch default?

Reply:
We can ignore the java and objective c files since they will be override in the next PRs